website/src/components/webinars/WebinarLayout.tsx

```diff
@@ -96,7 +96,11 @@ export const WebinarLayout: FunctionComponent<Props> = ({
 
                     <div className="col-md-6 col-12 pb-md-0 pb-6">
                         <h3 className="font-weight-bold">Watch the on-demand webinar</h3>
-                        <div className={`${isMdScreen || isMobile ? 'border-plum-mist' : styles.saturnBorder} border border-3 shadow-sm py-4 px-4 mx-1 mt-3`}>
+                        <div
+                            className={`${
+                                isMdScreen || isMobile ? 'border-plum-mist' : styles.saturnBorder
```

Comment:
Small nit/suggestion: to simplify this, you could use one condition like `isLgDown` to compute `windowWidth < breakpoints.lg` to apply the conditional style. I consider mobile starting at `0px` even though for accessibility we're supporting down to `320px`. However, I would vouch to just apply the `styles.saturnBorder` for consistency through all breakpoints even though the Figma shows a different border for mobile and desktop. Seems a bit overly complicated for a subtle border style change.

Reply:
Agreed, the `plum-mist` border is a bit over complex perhaps. I had some logo sizing styling for medium screens that differed from mobile, but I'll just combine them to reduce complexity too! Take a look now